content/webapp/pages/covid-welcome-back.tsx

```diff
@@ -125,8 +154,8 @@ const CovidWelcomeBackPage = () => {
                   </li>
                   <li>‘Medicine Man’ will be opening from 10 November.</li>
                   <li>
-                    The shop and Wellcome Kitchen are closed to make space for
-                    more café seating.
+                    Installations ‘Standardised Patient’ and ‘The Den’ will be
```

Comment:
should these link to the installations?

Reply:
I'll check (although Medicine Man above isn't, so possibly not)

Reply:
Checked with Kate and will add a link for all of them.